isso/js/embed.js

```diff
@@ -105,6 +112,9 @@ function fetchComments() {
             if (window.location.hash.length > 0 &&
                 window.location.hash.match("^#isso-[0-9]+$")) {
                 $(window.location.hash).scrollIntoView();
```

Comment:
Wrap this in a try-catch block (ES5 has it, I think?) or check if `$(window.location.hash` actually returns an element. `fetchComments` (and `init`) is a function that definitely should not throw.

Reply:
Yep, can't believe I didn't catch this error when I was testing. (It happens if you click a comment for the first time, before another one was linked, meaning there is no .isso-target)